test/stringinterface-test.jl

```diff
@@ -0,0 +1,14 @@
+function test_stringinterface()
+    print("stringinterface...")
+    Singular.libSingular.call_interpreter("ring r;")
+    result = Singular.libSingular.call_interpreter("r;")
+    @test result[1] == false
+    Singular.libSingular.call_interpreter("poly f = x;")
+    result = Singular.libSingular.call_interpreter("f;")
+    @test result[1] == false
+    @test result[2] == "x\n"
+    @test result[3] == ""
+    @test result[4] == ""
+    @test length(result) == 4
```

Comment:
I have a feel that this test would be a lot more helpful if it was closer to a GAP style `.tst` test. Luckily, as I learned recently in Kaiserslautern, this can be done in Julia, too, using a `jldoctest` (see <https://juliadocs.github.io/Documenter.jl/stable/man/doctests/>), which at the same time put a nice example into the manual, which is always good, isn't it ? :-)

Reply:
I would really prefer to not document this function in the manual. It should not be used, as it behaves absolutely orthogonal to the rest of Singular.jl, and might cause the user more headache than it serves them. This is mostly to be used by @mohamed-barakat , and I do not even think that his use-case is useful.